### PR TITLE
[18.0][MIG] cetmix_tower_yaml

### PR DESCRIPTION
**Pull request by ivs-cetmix**
Standard migration to 18.0

Task: 5104

## Summary by CodeRabbit

* **New Features**
  * Full YAML import/export for Cetmix Tower with nested relations, manifest support, UI export tabs/buttons, and manifest templates/authors management
  * Export wizard (file generation, filename sanitization, manifest options) and download flow
  * Import wizards (upload, validation, preview, create/update/skip flows)

* **Security**
  * New groups to control YAML import and export access

* **Tests**
  * Extensive test suites covering export/import, mixin behavior, round-trips and edge cases

* **Documentation**
  * README, usage, configure and static HTML docs added

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

### CODE REVIEW

File: cetmix_tower_yaml/models/cx_tower_server_log.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class CxTowerServerLog(models.Model):
+    _name = "cx.tower.server.log"
+    _inherit = [
+        "cx.tower.server.log",
+        "cx.tower.yaml.mixin",
+    ]
```
[the remark this answers is not in the document]
Comment (ivs-cetmix) [reply cetmix_tower_yaml/models/cx_tower_server_log.py:11]: You must provide a name when you are inheriting multiple models to specify the target model

File: cetmix_tower_yaml/models/cx_tower_yaml_manifest_template.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+        The version must consist of three non-negative integers (major, minor, patch)
+        separated by dots—for example, “1.2.3”. Raises a ValidationError otherwise.
+        """
+        semver = re.compile(r"^\d+\.\d+\.\d+$")
+        for rec in self:
+            if not semver.match(rec.version):
+                raise ValidationError(
+                    _("Version must be in the Major.Minor.Patch format, e.g. 1.2.3")
+                )
```
[the remark this answers is not in the document]
Comment (ivs-cetmix) [reply cetmix_tower_yaml/models/cx_tower_yaml_manifest_template.py:93]: Version is not a mandatory field

File: cetmix_tower_yaml/i18n/hr.po [hunk 3]
```diff
@@ -0,0 +1,587 @@
...
+#. module: cetmix_tower_yaml
+#: model:ir.model,name:cetmix_tower_yaml.model_cx_tower_yaml_import_wiz_upload
+msgid "Cetmix Tower YAML Import Wizard Upload"
+msgstr ""
+
+#. module: cetmix_tower_yaml
+#: model:ir.model,name:cetmix_tower_yaml.model_cx_tower_yaml_mixin
+msgid "Cetmix Tower YAML rendering mixin"
+msgstr ""
+
```
[the remark this answers is not in the document]
Comment (ivs-cetmix) [reply cetmix_tower_yaml/i18n/hr.po:145]: Will be added later

**Review by Aldeigja — APPROVED** [review]
Functional LGTM

**Review by GabbasovDinar — APPROVED** [review]

**Comment by CetmixGitDrone** [comment]
Hey, thanks for contributing! Proceeding to merge this for you.
Prepared branch [18.0-ocabot-merge-pr-432-by-ivs-cetmix-bump-nobump](https://github.com/cetmix/cetmix-tower/commits/18.0-ocabot-merge-pr-432-by-ivs-cetmix-bump-nobump), awaiting test results.

**Comment by CetmixGitDrone** [comment]
Congratulations, your PR was merged at fb273946b06e1316310b57cf4bd7d905b8d49f06. Thanks a lot for contributing to cetmix. ❤️
